File: DMS-main/DMS_API/DamandMgmt.Domain/Services/IEmployeeService.cs
using DemandMgmt.Domain.Model.Employee;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DemandMgmt.Domain.Services
{
    public interface IEmployeeService
    {
        List<EmployeeModel> GetAllEmployees();

        List<EmployeeModel> GetAllEmployeesAssignedToUID(List<string> uids);

        (List<string>, bool) UpdateEmployeeAssignmentForApprovalProcess(int EMPID, int allocationStatus, string UID);

    }
}

File: DMS-main/DMS_UI/src/BusinessLogic/Services/Employee/IEmployeeService.ts
import { Employee } from "../../../Domain/Employee/Employee";
import { Service } from "../Service";

export interface IEmployeeService extends Service{
    GetAllEmployees(): Promise<Employee[]>;
    AssignEmployee(EmpID: number, Uid: string, assignEmployee: boolean): Promise<{status: boolean, errorList: string[]}>;
}